transform/mattermost-analytics/models/intermediate/product/boards_active_users/_int_boards_active_users__models.yml

```diff
@@ -0,0 +1,107 @@
+version: 2
+
+models:
+  - name: int_boards_client_telemetry_daily
+    description: |
+      User activity per date and server. Used to mark the days that a user was active on boards for a specific server.
+      Reported by front-end telemetry. This table contains rows only for the dates/servers that a user is active on.
+
+    columns:
+      - name: daily_user_id
+        description: A unique id for each date/server/user combination.
+      - name: activity_date
+        description: The date that the user was active on.
+      - name: server_id
+        description: The server's unique id.
+      - name: user_id
+        description: The user's unique id.
+      - name: is_active
+        description: Boolean flag marking whether the user was active on the specific date and server. Always true.
+      - name: received_at_date
+        description: |
+          Date that the event for the user activity was received. Used for incremental load only, not to be used
+          downstream.
+
+  - name: int_boards_client_active_days
+    description: |
+      User activity aggregated per day and server. Calculates daily, weekly and monthly active users based on telemetry.
+      This model aggregates data from model int_boards_client_telemetry_daily.
+
+    columns:
+      - name: daily_user_id
```

Comment:
@ifoukarakis , should this be `daily_server_id` instead? (sorry, missed in the initial review).

Reply:
Nice catch! Creating a hotfix as this PR has been merged.